matchzoo/models/match_lstm.py

```diff
@@ -0,0 +1,78 @@
+"""Match LSTM model."""
+import keras
+import keras.backend as K
+from matchzoo import engine, preprocessors
+
+
+class MLSTMModel(engine.BaseModel):
+    """
+    Match LSTM model.
+
+    Examples:
+        >>> model = MLSTMModel()
+        >>> model.guess_and_fill_missing_params(verbose=0)
+        >>> model.build()
+
+    """
+
+    @classmethod
+    def get_default_preprocessor(cls):
+        """Get default preprocessor."""
+        return preprocessors.NaivePreprocessor()
```

Comment:
`BaseModel.get_default_preprocessor` already returns `NaivePreprocessor`. However, if this is just a placeholder for future development, then it's fine.